dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -99,7 +99,7 @@ cpu.systemLoadAverage.description=Displays the system load average for the last
 cpu.processCpuLoad.description=Displays the 'recent cpu usage' for the Java Virtual Machine process.
 
 #Cpu processCpuTime message
-#cpu.processCpuTime.description=Displays the CPU time used by the process on which the Java Virtual Machine is running.
+cpu.processCpuTime.description=Displays the CPU time used by the process on which the Java Virtual Machine is running.
```

Comment:
Change
"Displays the CPU time used by the process on which the Java Virtual Machine is running."
To
"Displays the CPU time that is used by the process on which the Java virtual machine is running."

Reply:
Done, thanks!